tools/board_stubs/build_board_specific_stubs/board_stub_builder.py

```diff
@@ -0,0 +1,312 @@
+            pins.append(f"{board_member}: microcontroller.Pin{comment}")
+
+        elif board_type == "board_i2c":
+            needs_busio = True
+            import_name = "I2C"
+            class_name = f"busio.{import_name}"
+            member_data = f"def {board_member}() -> {class_name}:\n"
+            member_data += f'    """Returns the `{class_name}` object for the board\'s designated I2C bus(es).\n'
+            member_data += f"    The object created is a singleton, and uses the default parameter values for `{class_name}`.\n"
+            member_data += '    """\n'
+            members.append(member_data)
+
+        elif board_type == "board_spi":
+            needs_busio = True
+            import_name = "SPI"
+            class_name = f"busio.{import_name}"
+            member_data = f"def {board_member}() -> {class_name}:\n"
+            member_data += f'    """Returns the `{class_name}` object for the board\'s designated SPI bus(es).\n'
+            member_data += f"    The object created is a singleton, and uses the default parameter values for `{class_name}`.\n"
+            member_data += '    """\n'
+            members.append(member_data)
+
+        elif board_type == "board_uart":
+            needs_busio = True
+            import_name = "UART"
+            class_name = f"busio.{import_name}"
+            member_data = f"def {board_member}() -> {class_name}:\n"
+            member_data += f'    """Returns the `{class_name}` object for the board\'s designated UART bus(es).\n'
+            member_data += f"    The object created is a singleton, and uses the default parameter values for `{class_name}`.\n"
+            member_data += '    """\n'
+            members.append(member_data)
+
+        elif board_type == "displays":
+            needs_displayio = True
+            if extra == "display":
+                import_name = "Display"
+            elif extra == "epaper_display":
+                import_name = "EPaperDisplay"
+            class_name = f"displayio.{import_name}"
+            member_data = (
+                f'"""Returns the `{class_name}` object for the board\'s built in display.\n'
+            )
+            member_data += f"The object created is a singleton, and uses the default parameter values for `{class_name}`.\n"
+            member_data += '"""\n'
+            member_data += f"{board_member}: {class_name}\n"
+            members.append(member_data)
+
+        elif board_type == "unmapped":
+            unmapped.append(extra)
+
+    libraries = mappings["libraries"]
+    included_modules = "Unknown"
+    frozen_libraries = "Unknown"
+    if "modules" in libraries:
+        included_modules = ", ".join(libraries["modules"])
+    if "frozen_libraries" in libraries:
+        frozen_libraries = ", ".join(libraries["frozen_libraries"])
+
+    with open(board_filename, "w") as boards_file:
+        boards_file.write("# SPDX-FileCopyrightText: 2024 Justin Myers for Adafruit Industries\n")
```

Comment:
```suggestion
        boards_file.write("# SPDX-FileCopyrightText: 2024 Justin Myers\n")
```

I only add "for Adafruit" because I'm paid to do it.

Reply:
Interesting. I really did write it for you...